Dockerfile

```diff
@@ -165,6 +165,13 @@ COPY --from=build /usr/lib/postgresql/${PGVERSION}/bin/pg_autoctl /usr/local/bin
 RUN mkdir -p /var/lib/postgres \
  && chown -R docker /var/lib/postgres
 
+RUN mkdir -p /tmp/extended_a && chown -R docker /tmp/extended_a
+RUN mkdir -p /tmp/extended_b && chown -R docker /tmp/extended_b
+RUN mkdir -p /tmp/extended_c && chown -R docker /tmp/extended_c
+RUN mkdir -p /tmp/pg/data \
+    && chown -R docker /tmp/pg \
+    && chown -R docker /tmp/pg/data
+
```

Comment:
Is there a way to keep those directories outside of the main Dockerfile, and also, in another directory... maybe in /var/lib/postgres/tablespaces or something. About the Dockerfile, we could have a specific Dockerfile in the new test directory, and use `build: .` in the docker-compose file. The new Dockerfile could inherit (as in `FROM ...`) from the local image. I think `FROM pg_auto_failover` would do the trick.

Reply:
Yes, I was hoping we could clean this up a bit. Thanks for the guidance.

Reply:
I'll try this out now-- I was following what exists in the upgrade docker-compose

Reply:
Pushed a fix